docs/spark/how-to-guides/ubuntu-instructions.md

```diff
@@ -0,0 +1,231 @@
+
+The next step is to build the .NET for Apache Spark Scala extension layer.
+
+```bash
+cd src/scala
+mvn clean package 
+```
+
+You should see JARs created for the supported Spark versions:
+
+* `microsoft-spark-2.3.x/target/microsoft-spark-2.3.x-<version>.jar`
+* `microsoft-spark-2.4.x/target/microsoft-spark-2.4.x-<version>.jar`
+
+### Build .NET sample applications using .NET Core CLI
+
+  1. Build the worker,
+
+      ```dotnetcli
+      cd ~/dotnet.spark/src/csharp/Microsoft.Spark.Worker/
+      dotnet publish -f netcoreapp2.1 -r ubuntu.18.04-x64
+      ```
+      
+      Sample console output:
+
+      ```bash
+      user@machine:/home/user/dotnet.spark/src/csharp/Microsoft.Spark.Worker$ dotnet publish -f netcoreapp2.1 -r ubuntu.18.04-x64
+      Microsoft (R) Build Engine version 16.0.462+g62fb89029d for .NET Core
+      Copyright (C) Microsoft Corporation. All rights reserved.
+      
+        Restore completed in 36.03 ms for /home/user/dotnet.spark/src/csharp/Microsoft.Spark.Worker/Microsoft.Spark.Worker.csproj.
+        Restore completed in 35.94 ms for /home/user/dotnet.spark/src/csharp/Microsoft.Spark/Microsoft.Spark.csproj.
+        Microsoft.Spark -> /home/user/dotnet.spark/artifacts/bin/Microsoft.Spark/Debug/netstandard2.0/Microsoft.Spark.dll
+        Microsoft.Spark.Worker -> /home/user/dotnet.spark/artifacts/bin/Microsoft.Spark.Worker/Debug/netcoreapp2.1/ubuntu.18.04-x64/Microsoft.Spark.Worker.dll
+        Microsoft.Spark.Worker -> /home/user/dotnet.spark/artifacts/bin/Microsoft.Spark.Worker/Debug/netcoreapp2.1/ubuntu.18.04-x64/publish/
+      ```
+
+  2. Build the samples.
+
+      ```dotnetcli
+      cd ~/dotnet.spark/examples/Microsoft.Spark.CSharp.Examples/
+      dotnet publish -f netcoreapp2.1 -r ubuntu.18.04-x64
+      ```
+      
+      Sample console output:
+
+      ```bash
+      user@machine:/home/user/dotnet.spark/examples/Microsoft.Spark.CSharp.Examples$ dotnet publish -f netcoreapp2.1 -r ubuntu.18.04-x64
+      Microsoft (R) Build Engine version 16.0.462+g62fb89029d for .NET Core
+      Copyright (C) Microsoft Corporation. All rights reserved.
+
+        Restore completed in 37.11 ms for /home/user/dotnet.spark/src/csharp/Microsoft.Spark/Microsoft.Spark.csproj.
+        Restore completed in 281.63 ms for /home/user/dotnet.spark/examples/Microsoft.Spark.CSharp.Examples/Microsoft.Spark.CSharp.Examples.csproj.
+        Microsoft.Spark -> /home/user/dotnet.spark/artifacts/bin/Microsoft.Spark/Debug/netstandard2.0/Microsoft.Spark.dll
+        Microsoft.Spark.CSharp.Examples -> /home/user/dotnet.spark/artifacts/bin/Microsoft.Spark.CSharp.Examples/Debug/netcoreapp2.1/ubuntu.18.04-x64/Microsoft.Spark.CSharp.Examples.dll
+        Microsoft.Spark.CSharp.Examples -> /home/user/dotnet.spark/artifacts/bin/Microsoft.Spark.CSharp.Examples/Debug/netcoreapp2.1/ubuntu.18.04-x64/publish/
+      ```  
+
+## Run samples
```

Comment:
Would make this title more consistent with other titles (see my comment about title above/about build and run titles in the Window how-to doc).